cfgov/v1/models/learn_page.py

```diff
@@ -299,27 +299,29 @@ class EventPage(AbstractFilterPage):
                 FieldPanel("end_dt", classname="col6"),
             ]
         ),
+        FieldPanel("future_body", heading="Content visible before event"),
+        FieldPanel("live_body", heading="Content visible during event"),
         MultiFieldPanel(
             [
-                FieldPanel("archive_body"),
-                FieldPanel("archive_image"),
+                FieldPanel(
+                    "archive_body", heading="Content visible after event"
+                ),
+                FieldPanel("archive_image", heading="Image shown after event"),
                 FieldPanel("video_transcript"),
                 FieldPanel("speech_transcript"),
-                FieldPanel("flickr_url"),
+                # FieldPanel("flickr_url"),
```

Comment:
Any reason to leave this commented out, versus removing the flickr_url field from the page? Doing so would require a data migration but Django should be able to auto-generate it.

Reply:
I commented it out because I'm a little uncertain whether anyone cares about that functionality; I haven't seen it used in my time here but would defer to anyone who thought it could still be useful moving forward.

Reply:
There's a single page on the entire website with a value for this field, [this one](https://www.consumerfinance.gov/about-us/events/archive-past-events/june-2016-consumer-advisory-board-meeting/). Apparently there's [custom logic](https://github.com/cfpb/consumerfinance.gov/blob/1832b7edbe6e46a5e73b2901258fbafae8bd6cb6/cfgov/v1/jinja2/v1/events/_macros.html#L272-L282) that renders this:

![image](https://github.com/cfpb/consumerfinance.gov/assets/654645/31269d25-d945-4475-82d0-fca4eb5184c9)

We could edit that page to manually duplicate that content into its "Archive body".. there might be a similar calculation to make for the `speech_transcript` (completely unused) and `video_transcript` (only 2 pages: [1](https://www.consumerfinance.gov/about-us/events/archive-past-events/field-hearing-arbitration-albuquerque-nm/) (2016), [2](https://www.consumerfinance.gov/about-us/events/archive-past-events/cfpb-rental-assistance-finder-tool-webinar/), (2021)) fields which have similar custom rendering in that template:

![image](https://github.com/cfpb/consumerfinance.gov/assets/654645/c4b13b1c-4148-48bd-b615-ce9d46dde9e9)

All of these fields use [`<h4>`](https://github.com/cfpb/consumerfinance.gov/blob/1832b7edbe6e46a5e73b2901258fbafae8bd6cb6/cfgov/v1/jinja2/v1/events/_macros.html#L285) in the template which also causes an accessibility error now in Wagtail because of a missing heading level:

![image](https://github.com/cfpb/consumerfinance.gov/assets/654645/4db284b6-759a-4881-a230-060c1daebd98)

I'd propose deprecating all three of these fields in the interest of greater EventPage simplification.

Reply:
Migration added!

Reply:
Great! OK I think this is good to go once all the checks are complete, once you approve!